spec/board_spec.rb

```diff
@@ -37,20 +37,51 @@
   end
 
   describe '#valid_placement?' do
-    
-    it 'checks if the placement is equal to length of ship' do
+
+    it 'returns false if placement length is equal to length of ship' do
       board = Board.new
       cruiser = Ship.new("Cruiser", 3)
 
       expect(board.valid_placement?(cruiser, ["A1", "A2"])).to be false
     end
 
-    it 'checks if coordinates are consecutive' do
+    it 'returns true if placement length is equal to length of ship' do
+      board = Board.new
```

Comment:
Should we also be checking coordinates that are in decreasing consecutive order? I.e. ["D1", "C1", "B1"]

Reply:
Yep! Definitely should do this too.